tests/scripts/minikube.sh

```diff
@@ -63,6 +71,14 @@ case "${1:-}" in
     echo "updating the rook images"
     copy_image_to_cluster ${BUILD_REGISTRY}/rook-amd64 rook/rook:master
     copy_image_to_cluster ${BUILD_REGISTRY}/toolbox-amd64 rook/toolbox:master
+    # If current kubectl context is minikube, restart rook pods
+    if [ "$(kubectl config view 2>/dev/null | awk '/current-context/ {print $NF}')" = "minikube" ];
+    then
+        echo "Restarting Rook pods in \"rook\" namespace."
```

Comment:
can we add this as a separate command in the script? When I run `minikube update` that doesn't always mean I want to kill all the rook pods. I usually just want to restart certain pods. So maybe we have a new command such as `minikube restart`

Reply:
`update` is run to update images, I can't think of a scenario why a user would not want pods to use the new image, why running the update then? Adding a new command adds complexity, I think we should rather avoid that.

Reply:
i would actually expect the cluster to break if you do this for all pods. The mons will likely all be dead since `dataDirHostPath` is not set by default, and you will lose your osd data. So I think dev still needs to control the pods that are updated.